src/metabase/task.clj

```diff
@@ -328,3 +328,12 @@
        (catch Exception e#
          (log/error e# msg#)
          (throw (JobExecutionException. msg# e# true))))))
+
+(defmacro defjob
+  "Like `clojurewerkz.quartzite.task/defjob` but with a log context."
+  [jtype args & body]
+  `(defrecord ~jtype []
+     org.quartz.Job
+     (execute [this# ~@args]
+       (log/with-context {:quartz-job-type (quote ~jtype)}
+         ~@body))))
```

Comment:
It seems like you should be able to just delegate to `jobs/defjob` here with an ignore kondo comment.

Would prefer doing that as we will still track any changes made to Quartzite.